ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/toolbars/Bundle.properties

```diff
@@ -0,0 +1,4 @@
+mainWindow.swing.firmware.toolbarTitle = Firmware selection toolbar
```

Comment:
I've been experimenting more with this and in the way the LocalizingService is implemented it can't be used together with the ActionRegistration-annotation unless we make some compromises.

As I see it there are three options:
1. Let the actions be registered using the netbeans api which will require an extra localization bundle with default texts (which can be overrided with the MessageBundles) in the same way I've experimented with. The greatest benefit that I see it is that we will get autodiscovery of actions and their configuration (no need to manually register them like in MacroService, JogActionService, RunActionService). We will also get a lot less code to maintain and get more features like scaling icons and the translation of action names etc. The disadvantages is that the configuration is more spread out and if we wish to disable an action we need to override the configuration in layers.xml. Also the documentation for Netbeans platform is not the best (I needed a couple of days to figure out how all this worked...).
2. Continue to use ActionRegistrationService and let all translations be handled by LocalizationService. The greatest benefit is that all actions are registered in one place, one configuration to rule them all and we are in the control of the process. The greatest disadvantage is that we need to register our actions through services that we need to create. We would also need (fairly) complex logic to determine that an actions no longer exist and remove its instance file from the file system.
3. We mix them. Scratch the changes that I've experimented with and let the core functionality use the action registration and localization as it's implemented today. Plugin developers can use netbeans platform api and use it's standard autodiscovery functionality, etc. The disadvantages with this I think is that the way we implement things would be inconsistent, there would be no good recipe in how to create a plugin. Also, the only way to disable core functionality would be to make code changes.